.github/workflows/ci.yml

```diff
@@ -10,18 +10,62 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: NPM cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: macos-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            macos-node-
 
       - name: Install npm dependencies
-        run: |
-          npm install
+        run: npm install
       
       - name: Run lint
-        run: |
-          npm run lint
+        run: npm run lint
 
       - name: Run test
-        run: |
-          npm run test
+        run: npm run test
+
+      - name: Install JDK 13
+        uses: actions/setup-java@v1
+        with:
+          java-version: 13
```

Comment:
why not 14?

Reply:
using the same version of java as locally